add note about direnv allow (#2845)
wireapp/wire-server: docs/README.md

This folder contains another `.envrc` file that adds all the binaries needed to
build the docs to `$PATH`.

In short, when you `cd` into this folder, you should see this message:

```sh
direnv: error wire-server/docs/.envrc is blocked. Run `direnv allow` to approve its content
```

> Run `direnv allow` to allow the `.envrc` file to modify your environment, and you should then have all the binaries/correct environment needed to build the docs.

Reviewer: ```suggestion
Run `direnv allow` to allow the `.envrc` file to modify your environment. Then, you should have everything (binaries, environment variables) needed to build the docs.
```

Reviewer 2: Should we mention cachix here as well? Without it set up, `direnv allow` will take hours to complete.

Author: @elland do you have a link/recommendation of what to say about cachix?

Reviewer 2: It's mentioned here https://github.com/wireapp/wire-server/blob/develop/docs/src/how-to/install/dependencies.rst :)
